radio/src/translations/untranslated.h

```diff
@@ -129,5 +129,5 @@
 #define LEN_DSM_PROTOCOLS              "\004"
 #define TR_DSM_PROTOCOLS               "LP45""DSM2""DSMX"
 
-#define LEN_MULTI_PROTOCOLS            "\006"
-#define TR_MULTI_PROTOCOLS             "FlySky""Hubsan""FrSky\0""Hisky\0""V2x2\0 ""DSM\0  ""Devo\0 ""YD717\0""KN\0   ""SymaX\0""SLT\0  ""CX10\0 ""CG023\0""Bayang""ESky\0 ""MT99XX""MJXq\0 ""Shenqi""FY326\0""SFHSS\0""J6 PRO""FQ777\0""Assan\0""Hontai""OpnLrs""FS 2A\0""Q2x2\0 ""Walk.\0""Q303\0 ""GW008\0""DM002\0""Cabell""Esy150""H8 3D\0""Corona""CFlie\0""Hitec\0""WFly\0 ""Bugs\0 ""BugMin""Traxas""NC1701""E01X\0 ""V911S\0""GD00X\0""V761\0 ""KF606\0""Redpin""Potens""ZSX\0  ""FLYZON"
+#define LEN_MULTI_PROTOCOLS            "\007"
```

Comment:
I'm not sure you want to expand the Protocol to seven characters as the space in the X9D+ screen is limited to six? Not sure on the other Radios if this is also limited? Please review before changing.
Thanks - Rich

Reply:
I see this comment too late. It needs to be checked on 128x64 board, perhaps I pressed the button too quickly